app/src/main/java/org/greenstand/android/TreeTracker/models/Locations.kt

```diff
@@ -143,8 +143,8 @@ class LocationDataCapturer(
     private var locationsDeque: Deque<Location> = LinkedList()
     var generatedTreeUuid: UUID? = null
         private set
-    private var lastConvergenceWithinRange: Convergence? = null
-    private var currentConvergence: Convergence? = null
+    var lastConvergenceWithinRange: Convergence? = null
+    var currentConvergence: Convergence? = null
```

Comment:
This class was working fine without these being exposed. Can we find a way to fix the issue without exposing internal values?

Reply:
It needs to be exposed so as to be used in the TreeCapturer.kt file to check if locations were captured at all after tree tracking. I added a private setter property to it in a newer commit, so it can only be read and not modified from outside the Locations.kt file.